formatTest/unit_tests/input/wrappingTest.re

```diff
@@ -1946,3 +1946,16 @@ let tryingTheSameInLocalScope = {
   let blah = fun a b => a;       /* Done */
   let blah a b => a;             /* Done (almost) */
 };
+
+
+/**
+   Issue 940: https://github.com/facebook/reason/issues/940
+   The parens in the exception match case are required for correct parsing
+   i.e. (Sys_error _ as exc) instead of Sys_error _ as exc
+   The latter doesn't type-check with Error: Unbound value exc.
+   Warning 11 (unused match case) is also triggered.
+ */
```

Comment:
It would be nice to see a *non* aliased exception match as well. This is also something that should probably done in `typeCheckedTests` to doubly verify that we are parsing/printing correctly.